src/Propel/Generator/Command/Helper/DialogHelper.php

```diff
@@ -49,13 +60,57 @@ private function formatQuestion($question, $default = null)
         }
     }
 
-    public function select(OutputInterface $output, $question, $choices, $default = null, $attempts = false, $errorMessage = 'Value "%s" is invalid', $multiselect = false)
+    public function select($question, $choices, $default = null, $attempts = false, $errorMessage = 'Value "%s" is invalid', $multiselect = false)
+    {
+        return parent::select(  $this->output,
```

Comment:
@mpscholten , i aligned arguments because it's multiline constructor. Do you insist?